info-compiler/src/test/java/com/google/sps/InfoCompilerTest.java

```diff
@@ -141,9 +178,20 @@ public void queryCivicInformation_succeedWithMockResponse() throws Exception {
     when(httpClient.execute(anyObject(), argumentCaptor.capture())).thenReturn(ELECTION_RESPONSE);
     JsonObject json =
         InfoCompiler.requestHttpAndBuildJsonResponse(httpClient, httpGet);
+        //The response given by api for testing is missing a property
+        JsonObject mockElection = (JsonObject) json.getAsJsonArray("elections").get(0);
+        mockElection.addProperty("ocdDivisionId", "ocd-division/country:us/state:ny");
     assertThat(json).isEqualTo(electionJson);
   }
 
+  @Test
+  public void getIncumbents_checkWithMockJson() throws Exception {
```

Comment:
The test name should include what condition you are checking, not how you are checking it